src/Renraku/ReProperMethodProtocolNameRuleForInstanceCreation.class.st

```diff
@@ -0,0 +1,14 @@
+"
+Rule to favour 'instance creation' method protocol over similar ones (like 'instance-creation' or other)
+"
+Class {
+	#name : #ReProperMethodProtocolNameRuleForInstanceCreation,
+	#superclass : #ReProperMethodProtocolNameRule,
+	#category : #'Renraku-Rules'
+}
+
+{ #category : #idioms }
+ReProperMethodProtocolNameRuleForInstanceCreation >> protocolIdiom [ 
+
+	^self use: 'instance creation' insteadOf: #('instance-creation' 'instances-creation' 'instances creation')
+]
```

Comment:
@astares Probably a problem here.

Reply:
What kind of problem?

Reply:
Oh sorry, I read the title too fast. No problem in fact :)